sale_delivery_address_modify/i18n/sale_delivery_address_modify.pot

```diff
@@ -0,0 +1,28 @@
+# Translation of Odoo Server.
```

Comment:
Remove the pot file, as this is no longer needed and can confuse people if it's not up to date.